spec/unit/provider/mount/windows_spec.rb

```diff
@@ -106,9 +106,17 @@ class Volume
         @provider.mount_fs
       end
 
-      it "should not mount the filesystem if it is mounted" do
-        expect(@vol).not_to receive(:add)
-        allow(@current_resource).to receive(:mounted).and_return(true)
+      it "should remount the filesystem if it is mounted and the options have changed" do
+        @vol.should_receive(:add).with(:remote => @new_resource.device,
+                                       :username => @new_resource.username,
+                                       :domainname => @new_resource.domain,
+                                       :password => @new_resource.password)
+        @provider.mount_fs
+      end
+
+      it "should not mount the filesystem if it is mounted and the options have not changed" do
+        @vol.should_not_receive(:add)
+        @current_resource.stub(:mounted).and_return(true)
```

Comment:
Replace above with `allow(@current_resource).to receive(:mounted).and_return(true)`